src/main/app/utils/stringUtils.ts

```diff
@@ -4,4 +4,8 @@ export default class StringUtils {
     return !(value && value.length > 0)
   }
 
+  static toUpperCase (value: any): string {
+    return value ? value.toUpperCase() : value
```

Comment:
```
     return value ? value.toUpperCase() : ''
```

or you can't tell this method is always returning string (as you claim in line 7).

:)

Reply:
have updated it to expect a string for value as the parameter

Reply:
but if you expect to be a string it means you can use:
https://www.w3schools.com/jsref/jsref_touppercase.asp
http://www.c-sharpcorner.com/UploadFile/5089e0/string-object-method-in-typescript-part-7/

which means this method is not needed.

Reply:
true! hadn't even realised that. just copied it from else where ha. will remove it